pkg/urbit/vere/io/fore.c

```diff
@@ -69,6 +69,32 @@ _fore_inject(u3_auto* car_u, c3_c* pax_c)
   u3z(ovo);
 }
 
+/* _fore_import(): form an ovum from jammed archive at [pax_c] and inject it.
+*/
+static void
+_fore_import(u3_auto* car_u, c3_c* pax_c)
+{
+  // With apologies
+  u3_noun arc = u3ke_cue(u3m_file(pax_c));
```

Comment:
Both `u3m_file()` and `u3ke_cue()` can bail. It'd be nice to catch those and print a clean error message, but it's probably not urgent.